src/util/rules.ts

```diff
@@ -118,3 +118,43 @@ export function isValidCreditCardExpirationDate(
 
     return true
 }
+
+/**
+ * Checks whether the input is undefined, an empty string, or a valid VAT identification number (VATIN).
+ * @param input - The string to be validated.
+ * @returns Returns true if the input is undefined, an empty string, or a valid VATIN. Otherwise, returns a string error message indicating the validation failure.
+ */
+export function isUndefinedOrEmptyStringOrValidVATIN(
+    input: string
```

Comment:
the docs say this can be undefined, however the type says sth else

Reply:
Good point...

Reply:
00fb73f28089e2bd0ced3fa889dac139e14d6745